gems/sorbet-runtime/test/types/abstract_validation.rb

```diff
@@ -494,6 +494,7 @@ def foo(**opts); end
 
     parent = Class.new do
       def initialize(foo:)
+        super()
```

Comment:
```suggestion
        super
```

Reply:
Oh amazing, I had forgotten that `super` will forward args, but `super()` will not. Sorry for the bad suggestion!